app/src/main/res/values-es/strings.xml

```diff
@@ -223,21 +222,22 @@
     <string name="recalibrate_goiv_notification">Recalibrar</string>
     <string name="title_activity_ocr_calibration_result">Resultado calibración</string>
     <string name="title_activity_ocr_calibration_error">Error calibración</string>
+    <string name="recalibration_description">La recalibración mejora el rendimiento de GoIV mediante la búsqueda dinámica de posiciones de escaneo óptimas para su teléfono.</string>
     <string name="scan_optimization_notice_text">No has ejecutado la optimización de escaneo. Se recomienda hacer una optimización de escaneo en todos los dispositivos.</string>
     <string name="scan_optimization_screen_ratio_warning">GoIV ha detectado que tu pantalla no es estándar. Es posible que necesite una calibración de escaneo para que GoIV funcione correctamente.</string>
     <string name="scan_optimization_outdated_warning">La optimización de escaneo está desactualizada. ¡Realice otra calibración de escaneo para disfrutar de las últimas mejoras!</string>
     <string name="touch_screen_quick_iv_scan">Toca pantalla para reprevisualizar IV</string>
     <string name="discard_unsaved_changes">Hay cambios sin guardar.\n¿Estás seguro de descartar los cambios?</string>
     <string name="configuration_saved">¡Configuración guardada!</string>
     <string name="no_token_selected">Ninguna ficha seleccionada</string>
-    <string name="token_max_evolution">%1$s\n\nEsta ficha es una variante de máx. evolución, significa que devolverá el resultado como si su monstruo ya estuviera evolucionado al máximo, lo que podría ser interesante en muchos casos.</string>
+    <string name="token_max_evolution">%1$s  Esta ficha es una variante de máx. evolución, significa que devolverá el resultado como si su monstruo ya estuviera evolucionado al máximo, lo que podría ser interesante en muchos casos.</string>
```

Comment:
Please note that (due to the Android Studio translation editor) the newlines (\n) are missing: please reintroduce them.
I noticed the same error in other translation (and probably did the same error myself), we have to keep an eye on this!

Reply:
ok, I made this change because I saw you doing it in the italian translation xD